cmd/fyne_reactive/fx.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"fyne.io/fyne"
+	"fyne.io/fyne/canvas"
+	"fyne.io/fyne/dataapi"
+	"fyne.io/fyne/theme"
+	"github.com/fyne-io/examples/img/icon"
```

Comment:
Let's not have Fyne depend on our examples ;)

Reply:
good catch, yes.  Replaced with a built-in icon to make the dependency go away